src/common/copilot/telemetry/copilotTelemetry.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import { ITelemetry } from "../../../client/telemetry/ITelemetry";
+
+// Telemetry Event Names
+export const CopyCodeToClipboardEvent = 'CopyCodeToClipboardEvent';
+export const InsertCodeToEditorEvent = 'InsertCodeToEditorEvent';
+export const UserFeedbackThumbsUpEvent = 'ThumbsUpEvent';
+export const UserFeedbackThumbsDownEvent = 'ThumbsDownEvent';
+export const CopilotResponseSuccessEvent = 'CopilotResponseSuccessEvent';
+export const CopilotResponseFailureEvent = 'CopilotResponseFailureEvent';
+export const CopilotLoginSuccessEvent = 'CopilotLoginSuccessEvent';
+export const CopilotLoginFailureEvent = 'CopilotLoginFailureEvent';
+export const CopilotClearChatEvent = 'CopilotClearChatEvent';
+export const UserFeedbackSuccessEvent = 'UserFeedbackEventSuccessEvent';
+export const UserFeedbackFailureEvent = 'UserFeedbackEventFailureEvent';
+
+interface IProDevCopilotTelemetryData {
```

Comment:
create a model file and keep the interface there

Reply:
done